lib/fizzy/cxx20/numeric.hpp

```diff
@@ -0,0 +1,37 @@
+// Fizzy: A fast WebAssembly interpreter
+// Copyright 2019-2020 The Fizzy Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#if __cplusplus > 201703L
```

Comment:
This will not work, because you may be building in C++20, but without `<bit>` implemented.
Use [`__cpp_lib_bitops`](https://en.cppreference.com/w/cpp/feature_test#Library_features).

Reply:
Should use `__cpp_lib_span` for span too. And likewise https://github.com/wasmx/fizzy/pull/542#issuecomment-696691041 could be merged.

Reply:
Correct, although the `std::bit_cast` is similarly untestable.